### PR TITLE
README.md PoC

### PR DESCRIPTION
**Pull request by carrodher**
- There are two common sections across charts ("Rolling VS Immutable tags" and "Production configuration and horizontal scaling"). The first one is present in all the charts, while the second one is present when the chart contains a `values-production.yaml`. As both sections are very usual, I placed just after the table.
- After those common sections, there are other sections specific per chart.
- Avoid CLI commands when possible
```
.
├── Parameters
│  └── (Table)
└── Configuration and installation details
   ├── Rolling VS Immutable tags
   ├── Production configuration and horizontal scaling
   └── Forwarding the logs to another service
```

### CODE REVIEW

File: bitnami/fluentd/README.md [hunk 1]
```diff
@@ -234,39 +260,3 @@ $ helm install bitnami/fluentd \
     --set aggregator.extraEnv[1].name=ELASTICSEARCH_PORT \
     --set aggregator.extraEnv[1].value=your-port-here \
 ```
```
Comment (carrodher) [inline bitnami/fluentd/README.md]: IMHO it's interesting to maintain those CLI commands to show users how to use this configmap

File: bitnami/fluentd/README.md [hunk 2]
```diff
@@ -234,39 +260,3 @@ $ helm install bitnami/fluentd \
...
-```
-
-To horizontally scale this chart once it has been deployed:
-
-```console
-$ helm upgrade my-release bitnami/fluentd \
-  -f ./values-production.yaml \
-  --set statefulset.replicaCount=3
-```
-
```
Comment (carrodher) [inline bitnami/fluentd/README.md]: Replaced commands by text

File: bitnami/fluentd/README.md (hunk 1 above)
Comment (beltran-rubo) [reply bitnami/fluentd/README.md]: I would remove the command and mention the parameters the user needs to configure.

File: bitnami/fluentd/README.md [hunk 3]
```diff
@@ -140,6 +140,32 @@ $ helm install --name my-release -f values.yaml bitnami/fluentd
...
+
+- Enable prometheus to access fluentd metrics endpoint:
+```diff
+- metrics.enabled: false
++ metrics.enabled: true
+```
+
+To horizontally scale this chart once it has been deployed you can upgrade the deployment using a new value for the `aggregator.replicaCount` parameter.
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
```
Comment (beltran-rubo) [inline bitnami/fluentd/README.md]: I would move this one to the first in the "Configuration and installation details" as it is included to all of them

**Pull request merged**